packages/theia-extension/cdtcloud/src/node/compilation-service.ts

```diff
@@ -0,0 +1,59 @@
+import { injectable } from "inversify";
+import { CompilationService } from "../common/protocol";
+import { readdir } from 'fs/promises';
+import { createReadStream } from 'fs';
+import { RPCClient } from "./rpc-client";
+import 'reflect-metadata';
+import FormData from 'form-data';
+import { join } from 'path';
+import got from "got";
+
```

Comment:
Import for comment below

```suggestion
import * as Path from "path";

 ```